docs/flow/modin/backends/pyarrow/query_compiler.rst

```diff
@@ -0,0 +1,18 @@
+PyArrow Query Compiler
+""""""""""""""""""""""
+:py:class:`~modin.backends.pyarrow.query_compiler.PyarrowQueryCompiler` is responsible for compiling efficient
+DataFrame algebra queries for the :doc:`PyarrowOnRayFrame </flow/modin/experimental/engines/pyarrow_on_ray>`, 
+the such frames that are backed by ``pyarrow.Table`` objects.
```

Comment:
```suggestion
the frames for which are backed by ``pyarrow.Table`` objects.
```

Reply:
this probably should be "the frames which are backed by ``pyarrow.Table`` objects." (without *for*).
Changed it